WalletWasabi.Fluent/Views/Settings/GeneralSettingsTabView.axaml

```diff
@@ -11,37 +11,37 @@
   <StackPanel Classes="settingsLayout">
 
     <DockPanel>
-      <TextBlock Text="Dark Mode" />
+      <TextBlock Text="Dark mode" />
       <ToggleSwitch IsChecked="{Binding DarkModeEnabled}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Start Wasabi with the system" />
+      <TextBlock Text="Run Wasabi when computer starts" />
       <ToggleSwitch IsChecked="{Binding RunOnSystemStartup}" Command="{Binding StartupCommand}" />
     </DockPanel>
 
     <DockPanel>
-      <TextBlock Text="Hide Window on close" />
+      <TextBlock Text="Minimize Wasabi to system tray" />
```

Comment:
According to https://github.com/zkSNACKs/WalletWasabi/pull/6469#discussion_r722182838 & https://github.com/zkSNACKs/WalletWasabi/pull/6469#discussion_r722458644 we shouldn't say minimize.